### ISSUE TITLE
"method handler crashed" - Smart Contract Interaction Page

### ISSUE THREAD

**Issue by santosfilipe** [microsoft/vscode-azure-blockchain-ethereum#52]
I'm following the HelloBlockchain examples from [Microsoft's official documentation ](https://docs.microsoft.com/en-us/azure/blockchain/service/send-transaction), after launching the Smart Contract Interaction Page in VS Code to test the "SendRequest" method, once I provide a text entry for the message and hit "Execute", it fails with "`method handler crashed`", and no additional information is logged in the Output:

![image](https://user-images.githubusercontent.com/11950862/84548872-0b90a480-acdd-11ea-97e4-0f1851193a94.png)

The first thing that caught my eye is the "Requestor" field being empty, unlike the documentation. 

This is the `HelloBlockchain.sol` code that was automatically generated:
```
pragma solidity ^0.5.5;

contract HelloBlockchain
{
     //Set of States
    enum StateType { Request, Respond }

    //List of properties
    StateType public  State;
    address public  Requestor;
    address public  Responder;

    string public RequestMessage;
    string public ResponseMessage;

    event StateChanged(string stateData);

    // constructor function
    constructor(string memory message) public
    {
        RequestMessage = message;
        State = StateType.Request;

        emit StateChanged('Request');
    }

    // call this function to send a request
    function SendRequest(string memory requestMessage) public
    {
        Requestor = msg.sender;

        RequestMessage = requestMessage;
        State = StateType.Request;
    }

    // call this function to send a response
    function SendResponse(string memory responseMessage) public
    {
        Responder = msg.sender;

        // call ContractUpdated() to record this action
        ResponseMessage = responseMessage;
        State = StateType.Respond;

        emit StateChanged('Response');
    }
}
```

**Comment by caleteeter** [comment]
@santosfilipe Thanks for this, there are a few things here we can do better at.  First, it looks like you are attempting to use the Smart Contract Interaction with a contract deployed to Azure.  The interaction page currently only works with locally deployed contracts (ganache).   We have some changes needed to support the contracts on ABS (or Infura).

You bring up another good point, that I have been worried about as well, is that the debugging on the interaction page is terrible.  This is driven by the fact that this is essentially a react app running inside the IDE.  We need to get a better pipe wired up between this and the IDE for getting more runtime details (and debugging info).

**Comment by caleteeter** [comment]
We found this bug and resolved in #55
